crates/solver/src/arguments.rs

```diff
@@ -339,10 +343,18 @@ impl std::fmt::Display for Arguments {
         writeln!(f, "metrics_port: {}", self.metrics_port)?;
         writeln!(f, "max_merged_settlements: {}", self.max_merged_settlements)?;
         writeln!(f, "solver_time_limit: {:?}", self.solver_time_limit)?;
-        writeln!(
+        display_option(
             f,
-            "market_makable_token_list: {}",
-            self.market_makable_token_list
+            "market_makable_token_list",
+            &self.market_makable_token_list,
+        )?;
+        display_option(
+            f,
+            "market_makable_tokens",
+            &self
+                .market_makable_tokens
+                .as_ref()
+                .map(|list| format!("{list:?}")),
```

Comment:
Don't we only have tokens here?:
```suggestion
                .map(|token| format!("{token:?}")),
```

Reply:
Ahh, makes sense. 👌